gulpfile.js

```diff
@@ -1,40 +1,40 @@
-var gulp = require('gulp');
-
-var sass = require('gulp-sass'),
-  autoprefix = require('gulp-autoprefixer'),
-  jshint = require('gulp-jshint'),
-  stylish = require('jshint-stylish'),
-  notify = require('gulp-notify');
+var autoprefix = require("gulp-autoprefixer"),
+    bourbon    = require("bourbon").includePaths,
+    gulp       = require("gulp"),
+    jshint     = require("gulp-jshint"),
+    neat       = require("bourbon-neat").includePaths,
+    sass       = require("gulp-sass"),
+    stylish    = require("jshint-stylish");
+
+var paths = {
+  scss: "./assets/scss/**/*.scss",
```

Comment:
This recursive all `scss` files path seems good for the watch, but seems a bit weird for the entry currently on line 18 `return gulp.src(paths.scss)`. So that will compile 1:1 any `scss` file? don't you end up with a bunch of extra files in the destination?

Reply:
It still wont output any scss files prefixed with an underscore so it's fine.